<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="fw_taskBarLayout">
        <item>
         <widget class="QCheckBox" name="fw_activeCheckBox">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Active</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="activeClientStateLine">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>15</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="fw_clientStateComboBox">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>-</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Live</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>To File</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>To Ring File</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>To Shared File</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>To Parent</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>To Global</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="Line" name="clientStateNewestOnTopLine">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>15</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="fw_newestOnTopCheckBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Newest on Top</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="fw_scrollToBottomCheckBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Scroll To Bottom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="scrollToBottomTabModeLine">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>15</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="fw_tabModeRadio">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Tab Mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="fw_windowModeRadio">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Window Mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>15</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Search:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="fw_caseSensitiveCheckBox">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Case Sensitive</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fw_inputFilter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1650505</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>Filter expression here.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QGridLayout" name="mainContentGrid" rowminimumheight="0">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
         </layout>
        </widget>
        <widget class="QPlainTextEdit" name="fw_logDetailInput">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>5</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="fw_buttonBarLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="fw_exportButton">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Export</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="exportLine">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fw_freezeButton">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Freeze/Unfreeze</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="fw_freezeState">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>10</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <kerning>false</kerning>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="documentTitle">
           <string notr="true"/>
          </property>
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="lineWrapMode">
           <enum>QPlainTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="freezeUnfreezeLine">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fw_freezeAllButton">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Freeze/Unfreeze all</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_8">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fw_minimizeButton">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Minimize</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_9">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fw_minimizeAllButton">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Minimize all</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1116</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionMinimize"/>
    <addaction name="actionMinimize_All"/>
    <addaction name="separator"/>
    <addaction name="actionArrange_Windows"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear Table</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="text">
    <string>Minimize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionMinimize_All">
   <property name="text">
    <string>Minimize All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionArrange_Windows">
   <property name="text">
    <string>Arrange Windows</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>fw_activeCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>activeCheckBoxClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>-25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fw_newestOnTopCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>newestOnTopCheckBoxClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fw_scrollToBottomCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>scrollToBottomCheckBoxClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>-15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fw_tabModeRadio</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>tabModeRadioClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>-7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fw_windowModeRadio</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>windowModeRadioClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>782</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fw_exportButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>exportButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>647</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fw_freezeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>freezeButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>656</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fw_freezeAllButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>freezeAllButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>671</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>656</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fw_minimizeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>minimizeButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>888</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>696</x>
     <y>666</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fw_minimizeAllButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>minimizeAllButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1089</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>820</x>
     <y>665</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fw_inputFilter</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>filterInputTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>992</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>1002</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fw_inputFilter</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>inputFilterEnterPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1011</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>1030</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClear</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>clearTable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExport</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>exportButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMinimize</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>minimizeButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMinimize_All</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>minimizeAllButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionArrange_Windows</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>arrangeWindowsTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fw_clientStateComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>clientStateComboBoxChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>activeCheckBoxClicked()</slot>
  <slot>liveCheckBoxClicked()</slot>
  <slot>toFileCheckBoxClicked()</slot>
  <slot>toRingFileCheckBoxClicked()</slot>
  <slot>toParentCheckBoxClicked()</slot>
  <slot>newestOnTopCheckBoxClicked()</slot>
  <slot>scrollToBottomCheckBoxClicked()</slot>
  <slot>tabModeRadioClicked()</slot>
  <slot>windowModeRadioClicked()</slot>
  <slot>exportButtonClicked()</slot>
  <slot>freezeButtonClicked()</slot>
  <slot>freezeAllButtonClicked()</slot>
  <slot>minimizeButtonClicked()</slot>
  <slot>minimizeAllButtonClicked()</slot>
  <slot>filterInputTextChanged(QString)</slot>
  <slot>inputFilterEnterPressed()</slot>
  <slot>toSharedFileCheckBoxClicked()</slot>
  <slot>toGlobalCheckBoxClicked()</slot>
  <slot>clearTable()</slot>
  <slot>arrangeWindowsTriggered()</slot>
  <slot>clientStateComboBoxChanged(int)</slot>
 </slots>
</ui>
